docs/submit-transactions.md

```diff
@@ -0,0 +1,149 @@
+⚠ Before proceeding, make sure you have [installed the required binaries](getting-started#installation)
+and have followed the getting started steps regarding [submitting a transaction](getting-started#submitting-a-transaction).
+
+
+## Available transactions
+
+Discover all the available transactions that you can submit via the CLI.
+Run the provided help commands to learn more about the arguments
+and behaviour of each one of them.
+
+ⓘ All the transactions provide the same [Command-Line Options](#command-line-options).
+
+- Transfer funds from an account to another
+
+    ``` bash
+    radicle-registry-cli account transfer --help
+    ```
+
+- Transfer funds from an org's account to another account
+
+    ``` bash
+    radicle-registry-cli org transfer --help
+    ```
+
+- Register entities
+
+    You can register users, orgs, and projects in the Radicle Registry.
+
+    - Register a user
+        ``` bash
+        radicle-registry-cli user register --help
+        ```
+
+    - Register an org
+        ``` bash
+        radicle-registry-cli org register --help
+        ```
+
+    - Register a project
+        ``` bash
+        radicle-registry-cli project register --help
+        ```
+
+- Unregister entities
+
+    You can unregister orgs and users from the Radicle Registry.
+
+    - Unregister a user
+        ``` bash
+        radicle-registry-cli user unregister --help
+        ```
+
+    - Unregister an org
+        ``` bash
+        radicle-registry-cli org unregister --help
+        ```
+
+### Examples
+
+1. Register a user
+
+    This example registers a user named "neo" in the registry using a
+    locally generated account (that happens to also be) named "neo"
+    to sign the transactions, specifying `100` as the fee.
+
+    ``` bash
+    radicle-registry-cli user register neo --author neo --fee 100
+    ```
+
+2. Transfer funds from an account to another
+
+    This example transfers `500` tokens to the account with the address
+    `5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd`, specifying `20` as fee. A total of
+    `520` will be withdrawn from the author's account, neo.
+
+    ``` bash
+    radicle-registry-cli account transfer 500 5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd --author neo --fee 20
+    ```
+
+3. Unregister an org
+
+    This example unregisters an org named "Hooli". For it to succeed, such an org must be registered in
+the Radicle Registry and it must have been registered by the author. The author will be charged
+    `10` tokens in any case.
+
+    ``` bash
+    radicle-registry-cli org unregister Hooli --author neo --fee 10
+    ```
+
+## Command-Line Options
+
+All the CLI commands that submit transactions provide the following options that
+are to be specified at the end of your command.
+
+- `Author`
+
+    ⚠ Requires having [getting started submitting a transaction](getting-started#submitting-a-transaction).
+
+    Mandatory for all transactions, the author option lets you specify the local account
```

Comment:
We should clarify what the term “local account” means.

Reply:
I considered the same, but since a pre-requisite here is to follow `getting-started#submitting-a-transaction` where that is explained, I thought we could skip repeating that here. What do you recon?

Reply:
The section you’re referring to does tell a user how to create an account. But the term “local account” is never introduced. How about this.
```suggestion
    Mandatory for all transactions. The author option lets you specify the account that is used to sign
    the transaction. You can use the `account` command group to manage your accounts.
```

Reply:
That's good, will commit :+1: